ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml

```diff
@@ -970,6 +970,27 @@ tests:
   steps:
     cluster_profile: gcp-openshift-gce-devel-ci-2
     workflow: openshift-e2e-gcp-graceful-shutdown
+- as: e2e-agent-compact-ipv4
+  cluster: build05
```

Comment:
I would let the `make jobs` decide which build cluster to use.
You can remove this line.

Reply:
build05 is required to use OFCIR